src/action/index.ts

```diff
@@ -62,6 +67,17 @@ async function main(): Promise<void> {
   }
 }
 
+/**
+ * Set output with logging to stdout for test support
+ */
+function setOutput(...args: Parameters<typeof setOutputActionsCore>) {
```

Comment:
I think I needed this to make the tests backwards compatible with how the new setOutput works. If things are working without it now maybe we don't need it anymore.